ml-conversational-analytic-tool/test_commentAnalysis.py

```diff
@@ -0,0 +1,53 @@
+import unittest
+from commentAnalysis import CommentAnalyzer
+
+
+class TestCommentAnalysis(unittest.TestCase):
+
+    
+    def test_CommentAnalyzer_init(self):
+        analyzer = CommentAnalyzer(['test'])
+        """ positive test """
+        self.assertEqual(analyzer.word_count, {'test': 0})
```

Comment:
The syntax for assertEqual is (expected, actual). It's common to mistakenly have these swapped, but it matters when test failures are reported.

Reply:
Done